articles/references/concepts/project-layout.md

```diff
@@ -0,0 +1,58 @@
+# Project Layout (with Roles)
+
+## Overall layout
+
+Since we need to generate code at the end of the day, we need to tell Intent Architect where to place it.
+
+![Project Layout](images/project-layout/project-layout.png)
+
+This can be located by clicking on `Project Layout`, located on the left-hand panel.
+
+![Located](images/project-layout/locate-project-layout-button.png)
+
+You'll notice a gray folder called `Projects`. This is there by default. It cannot change.
+
+Inside that folder there are yellow folders. At the moment, they represent the virtual folders found in Visual Studio.
+
+Within those folders you may add different project types. In the sample image above you will notice a `Web project` type and a couple of `Class project` types.
+
+You may customize your solution to your hearts content by creating different folders and moving/modifying the projects already contained within.
+
+If you were to open up Visual Studio, you will find that the layout above corresponds with the Visual Studio tree layout:
+
```

Comment:
```suggestion
Within the the `Projects` root folder, projects and sub-folders can be created. Sub-folders can be created to any depth and projects can be placed at any level within your folder structure.

It is also possible to create and move `Roles`. Roles will be explained in full detail further below.

Modules can refer to this `Project Layout` configuration to determine where to output generated content. For example the `Intent.VisualStudio.Projects` module will generate a `.sln` which contains the folder structure matching the configuration here along with referencing and creating a `.csproj` for each project specified.
```

Additional suggestions (could be come back to later, perhaps we make GitHub issues to remind us?):
- Link to the Visual Studio Module
- Link to Modules